src/fs.cpp

```diff
@@ -242,7 +242,11 @@ void ofstream::close()
 }
 #else // __GLIBCXX__
 
+#if BOOST_VERSION >= 107700
+static_assert(sizeof(BOOST_FILESYSTEM_C_STR(*fs::path())) == sizeof(wchar_t),
```

Comment:
Wouldn't
```c++
static_assert(sizeof(*BOOST_FILESYSTEM_C_STR(fs::path())) == sizeof(wchar_t),
```
communicate the intent here better?

Reply:
Agree.

The [member access](https://en.cppreference.com/w/cpp/language/operator_member_access#Built-in_member_access_operators) operator has [higher precedence](https://en.cppreference.com/w/cpp/language/operator_precedence) than the [indirection operator](https://en.cppreference.com/w/cpp/language/operator_member_access#Built-in_indirection_operator).

Reply:
Agree!